### PR TITLE
[W-11326016] - Modal Close button

### PR DESCRIPTION
**Pull request by JoseRossiSF**
# Technical Document
[Technical Design Review - W-11326016 - NPSP](https://salesforce.quip.com/Ma5qAHoUbK25)
Change to use a lighting-button-icon instead of the button element that was being used

# Changes

# Work Item
[W-11326016](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000zhIOXYA2/view)

### CODE REVIEW

**Review by maurimart — APPROVED** [review]
LGTM

**Review by gerardorodriguezsf — CHANGES_REQUESTED** [review]
Please review the indentation (4 spaces) from the worked files.

**Review by gerardorodriguezsf — APPROVED** [review]
LGTM

**Review by voduyemi — CHANGES_REQUESTED** [review]

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -1897,6 +1897,14 @@
         <shortDescription>Text indicating the Recurring Donation would be cancelled</shortDescription>
         <value>This can take some time. Refresh the page to check the status. Contact us if you need help.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateRDStopDonationMessage</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text indicating the Recurring Donation is cancelled</shortDescription>
+        <value>The nonprofit won't process any future donations for this Recurring Donation.</value>
```
Comment (andrewyu-salesforce) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:1906]: `'` does not work in custom label. Please change it to `&apos;`

File: force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html [hunk 2]
```diff
@@ -210,5 +209,7 @@
     </div>
     <c-update-payment-method-modal is-bank-payment-allowed={allowACHPaymentMethod} current-record={currentRecord} open-update-payment-method={openUpdatePaymentMethod} onclose={handleClose}></c-update-payment-method-modal>
     <c-change-amount-or-frequency-modal is-initially-monthly-donation={isInitiallyMonthlyDonation} is-elevate-donation={isElevateDonation} fixed-installments-label={fixedInstallmentsLabel} day-of-month-field-label={dayOfMonthFieldLabel} default-record-type-id={defaultRecordTypeId} current-record={currentRecord} open-change-amount-or-frequency={openChangeAmountOrFrequency} onclose={handleClose}></c-change-amount-or-frequency-modal>
-    <c-stop-recurring-donation-modal current-record={currentRecord} open-stop-recurring-donation={openStopRecurringDonation} onclose={handleClose}></c-stop-recurring-donation-modal>
+    <template if:true={openStopRecurringDonation}>
+        <c-stop-recurring-donation-modal current-record={currentRecord} open-stop-recurring-donation={openStopRecurringDonation} onclose={handleClose}></c-stop-recurring-donation-modal>
```
Comment (andrewyu-salesforce) [inline force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html:213]: Ideally, the jest test should be updated to reflect the front end change.

Comment (JoseRossiSF) [reply force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html:213]: Thanks for the suggestion. It has been added

**Review by lparrott — APPROVED** [review]

**Review by andrewyu-salesforce — APPROVED** [review]
looks good

**Pull request merged**